buildSrc/src/main/groovy/VeracodeTask.groovy

```diff
@@ -100,7 +134,10 @@ abstract class VeracodeTask extends DefaultTask {
     }
 
     static class VeracodeUser {
```

Comment:
it might make sense to have 2 separate class, as this class contains 2 purposes, contain veracode api  credentials and user name credentials. We could also rename this class to something that expressive enough to convey information that it would contains credentials, api credentials of veracode and user